nixpkgs_review/report.py

```diff
@@ -67,7 +67,7 @@ def write_error_logs(attrs: List[Attr], directory: Path) -> None:
                 symlink_source.unlink()
             symlink_source.symlink_to(attr.path)
 
-        for path in [attr.drv_path, attr.path]:
+        for path in [f"{attr.drv_path}^*", attr.path]:
```

Comment:
What happens if people now run the program on nixos 23.05?

Reply:
works for me with nix 2.13.3 (`nixVersions.nix_2_13` on nixpkgs unstable)

Reply:
Just to confirm you also had the nix-daemon running on this version?

Reply:
oops, I didn't, I will do check with nix-daemon running nix 2.13

Reply:
I can confirm the `nix log` command still works after switching to nix 2.13 (including the daemon) on my machine